src/RadworksGovernor.sol

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {Governor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {RadworksGovernorTimelockCompound} from "src/lib/RadworksGovernorTimelockCompound.sol";
+import {GovernorCompatibilityBravo} from
+  "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
+
+/// @notice The upgraded Radworks Governor: Bravo compatible and extended from OpenZeppelin.
+contract RadworksGovernor is
+  Governor,
+  GovernorVotesComp,
+  RadworksGovernorTimelockCompound,
```

Comment:
I think we can use the stock ` GovernorTimelockCompound`. We used a custom Timelock contract for PoolTogether because their `GRACE_PERIOD` method was called `gracePeriod`. But, for Radicle it seems to be compatible.

Reply:
Looks like Radicle also calls it `gracePeriod` .. https://etherscan.io/address/0x8dA8f82d2BbDd896822de723F55D6EdF416130ba#code

Reply:
Thanks! Indeed.. I guess I missed that both were present: c7f59d45d8da2cc21cee3ad4d502f59eb9dfc04f